src/shapes/Polyline.ts

```diff
@@ -103,11 +100,36 @@ export class Polyline<
    */
   constructor(points: XY[] = [], options: Props = {} as Props) {
     super({ points, ...options });
-    const { left, top } = options;
+    const { left, top, fromSVG } = options;
     this.initialized = true;
-    this.setBoundingBox(true);
-    typeof left === 'number' && this.set('left', left);
-    typeof top === 'number' && this.set('top', top);
+    const {
+      left: calculatedLeft,
+      top: calculatedTop,
+      width,
+      height,
+      pathOffset,
+      strokeOffset,
+    } = this._calcDimensions();
+    this.set({ width, height, pathOffset, strokeOffset });
+    if (fromSVG) {
+      // if coming from SVG just assign top and left, those will be fixed
+      // later by remove removeTransformMatrixForSvgParsing using the weird
+      // _findCenterFromElement.
+      // This is not clear and is probably wrong, but as of now it works in conjuction
+      // with that other parsing function
+      this.set({
+        left: calculatedLeft,
+        top: calculatedTop,
+      });
+    } else {
+      // looking at out path behaves this is not correct.
+      // This should behave the same as path
+      this.setPositionByOrigin(
+        new Point(left ?? calculatedLeft, top ?? calculatedTop),
+        'left',
+        'top'
+      );
+    }
```

Comment:
had to unwind setBoundingBox in order to apply the correct code or i should have had setBoundingBox returning a point like path does, but if i remember correctly that is not something we want to do.

Reply:
We really not to sit down and look at open PRs
I believe I fixed this somewhere already